akeyless_gateway/README.md

```diff
@@ -0,0 +1,104 @@
+# Akeyless Gateway
+
+## Overview
+
+The Akeyless Platform is a unified secrets management system that enables you to store, protect, rotate, and dynamically create credentials, certificates, and encryption keys. Our platform supports several use cases, including managing static and dynamic credentials, certificate automation, encryption and digital signing, and zero-trust application access that secures remote access to your internal resources.
+
+This integration allows you to visualize and monitor performance of your [Akeyless Gateway](https://docs.akeyless.io/docs/api-gw). The Telemetry Metrics are based on time series telemetry data metrics from the application and the runtime environment, storing them in a unique database or index, and analyzing data trends over time.
+
+## Setup
+
+Akeyless offers a unique Gateway, which adds an extra level of protection between your private network and the cloud. Acting as a SaaS extension of our core services, our stateless Gateway enables a transparent internal operation with a robust out-of-the-box mechanism to ensure service continuity and recovery while you are not required to change any network infrastructure in order to work with your internal resources.
+
+To configure the integration with Datadog to view important Akeyless Gateway metrics, follow the below instructions for the method you are using (or have used) for your Gateway deployment.
+
+### Prerequisites
+- An Akeyless Gateway either running or being deployed for the first time
+
+### Configure
+
+This integration works with one Gateway or multiple instances using the same API key. Metrics can be shown per `host` or `instance` in the **Akeyless GW** dashboard.
+
+#### For a Gateway Running on Kubernetes
+
+To configure the Akeyless Gateway integration on a [Gateway running on K8s](https://docs.akeyless.io/docs/gateway-k8s):
+
+1. In your `values.yaml` file you use to deploy your Gateway on Kubernetes, under the `metrics` section, add the below configuration. Set the relevant API Key of your Datadog server, and the relevant site. If your Datadog server is running in the EU site, use `datadoghq.eu`. Default is `datadoghq.us`.
```

Comment:
```suggestion
1. In your `values.yaml` file you use to deploy your Gateway on Kubernetes, under the `metrics` section, add the below configuration. Set the relevant API Key of your Datadog server, and the relevant [Datadog site][1].
```
We're better off linking to this doc as there are a number of Datadog sites to support.

Reply:
I am happy with the link, but we want to be clear to our users what they need to actually add in the file. Let's please keep the last part: "The default server is `datadoghq.us`."

Reply:
Updated my suggestion. Feel free to commit any changes you'd prefer directly